.github/workflows/aks.yaml

```
@@ -1,6 +1,6 @@
 # This workflow calls the main workflow with custom variables
 name: AKS-E2E
-run-name: AKS ${{ inputs.downstream_k8s_minor_version }} on Rancher v${{ inputs.rancher_version || '2.9-head' }} deployed on ${{ inputs.k3s_version || 'v1.30.5+k3s1' }}
+run-name: AKS ${{ inputs.downstream_k8s_minor_version }} on Rancher ${{ inputs.rancher_version || 'latest/devel/2.9' }} deployed on ${{ inputs.k3s_version || 'v1.30.5+k3s1' }}
```

Perhaps it's time to move the default to 2.10?
```suggestion
run-name: AKS ${{ inputs.downstream_k8s_minor_version }} on Rancher ${{ inputs.rancher_version || 'latest/devel/2.10' }} deployed on ${{ inputs.k3s_version || 'v1.30.5+k3s1' }}
```